spec/controllers/v0/dependents_applications_controller_spec.rb

```diff
@@ -42,7 +42,8 @@
   describe 'POST create' do
     context 'with valid params v1' do
       before do
-        Flipper.disable(:va_dependents_v2)
+        allow(Flipper).to receive(:enabled?).with(:va_dependents_v2).and_return(false)
+        allow(Flipper).to receive(:enabled?).with(:remove_pciu, instance_of(User)).and_return(false)
```

Comment:
oh, nice! So adding the pciu flipper as disabled gets the tests to pass. But do you know why this would be needed? Are they connected in some way?

Reply:
Somewhere the dependents application controller is hitting FormProfile, which calls the remove_pciu flipper.